src/busco/test.sh

```diff
@@ -0,0 +1,43 @@
+test_dir="$meta_resources_dir/test_data"
+
+echo "> Running busco with lineage dataset"
```

Comment:
To make sure one run doesn't influence the other

```suggestion
mkdir "run_prot_stramenopiles"
cd "run_prot_stramenopiles"

echo "> Running busco with lineage dataset"
```